ecs-cli/modules/cli/compose/entity/service/service_helper.go

```diff
@@ -15,22 +15,76 @@ package service
 
 import (
 	"fmt"
+	"sort"
+	"time"
 
 	log "github.com/Sirupsen/logrus"
-	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/utils/waiters"
+	ecscli "github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands"
 	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
 )
 
+// UpdateServiceTimeout is the time that the CLI will wait to check if the
+// count of running tasks is changing. If it has not changed then an error is thrown
+// after UpdateServiceTimeout minutes
+const DefaultUpdateServiceTimeout = 5
+const latestEventWindow = 5
```

Comment:
Perhaps worth noting in a comment that the unit for `latestEventWindow` is seconds whereas for `DefaultUpdateServiceTimeout` is minutes?